spec/features/workflow/unwithdraw_without_managing_editor_permissions.spec.rb

```diff
@@ -2,12 +2,17 @@
 
 RSpec.feature "Undo a withdraw without managing editor permission" do
   scenario "withdrawn edition" do
+    given_i_am_not_a_managing_editor
```

Comment:
Similarly, do we actually need this step?

Reply:
I think this is helpful just to be really clear when reading the tests that the user must not be a managing editor.

Reply:
@emmabeynon yup the nice part is the clarity in the naming, the bummer though of my change was that the code was just `login_as(create(:user))` which lacks the clarity of the method.

Could presumably go down a route of `create(:user, :non_managing_editor)` but then the code for it is growing.